sda/cmd/auth/frontend/templates/oidc.html

```diff
@@ -28,29 +28,22 @@
     <div class="jumbotron" role="region">
       <div class="container" id="loginbox1">
             <p class="lead text-center">
-              Welcome, {{.User}}!
+              Welcome, {{.Fullname}}!
             </p>
-            {{if .Passport}}
+            {{if .ResignedToken}}
             <p class="text-center">
-              Your visas are the following:
+              For <strong>uploading</strong> to the Inbox, your access token {{if .ExpDateResigned}}(expires at {{.ExpDateResigned}} UTC){{end}} is:
             </p>
-            {{ range .Passport }}
-            <pre class="border border-secondary rounded py-2 px-3 my-4" id="logintext">{{.}}</pre>
-            {{ end }}
-            {{ end }}
-            {{if .Token}}
-            <p class="text-center">
-              Your access token is:
-            </p>
-            <pre class="border border-secondary rounded py-2 px-3 my-4" id="logintext">{{.Token}}</pre>
+            <pre class="border border-secondary rounded py-2 px-3 my-4" id="logintext">{{.ResignedToken}}</pre>
             {{end}}
-            {{if .ExpDate}}
+            <a href="/oidc/s3conf-inbox" class="btn btn-primary btn-block mb-5">Download credentials to upload to the Inbox</a>
```

Comment:
This should not be visible if the CEGA login is enabled.

Reply:
If I understand correctly, after CEGA login, the page will be [ega.html](https://github.com/neicnordic/sensitive-data-archive/blob/main/sda/cmd/auth/frontend/templates/ega.html), not oidc.html. 

Also when testing the EGA login, I found a bug at https://github.com/neicnordic/sensitive-data-archive/blob/main/sda/cmd/auth/main.go#L208 which makes `loginform.html` failed to render. There's also redundant code in this function which needs to be refactored. I guess it is better to be addressed in a new issue?

Reply:
In a F-EGA setup the ega-login is used for uploading data.
But in order to download data the user will have to authenticate using the LS-AAI login. And then the user should not see the 'upload credentials' section of the LS-login response page.